### PR TITLE
Fix incorrect "categories" and "name" fields in sample.json files.

### PR DESCRIPTION
**Pull request by xmnboy**
# Existing Sample Changes

## Description

Fix incorrect "categories" and "name" fields in sample.json files and cleaned up README.md files to make them more readable as plain text files.

## External Dependencies

N/A

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Implement fixes for ONSAM Jiras

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Command Line
- [X] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ ] VSCode
- [ ] When compiling the compliler flag "-Wall -Wformat-security -Werror=format-security" was used

FYI: @JoeOster and @tomlenth -- please merge into master to fix these two samples as they appear in the sample browser trees. Only `sample.json` and text files were updated. No source code or build files were touched.

### CODE REVIEW

**Review by JoeOster — APPROVED** [review]

File: DirectProgramming/C++/CompilerInfrastructure/OpenMP_Offload_Features/License.txt [hunk 1]
```diff
@@ -1,8 +1,19 @@
-Copyright Intel Corporation
+Copyright (c) Intel Corporation
```
Comment (JoeOster) [inline DirectProgramming/C++/CompilerInfrastructure/OpenMP_Offload_Features/License.txt:1]: Why the update? The (c) is not needed per legal

File: AI-and-Analytics/End-to-end-Workloads/Census/License.txt [hunk 2]
```diff
@@ -1,7 +1,19 @@
-Copyright Intel Corporation
+Copyright (c) Intel Corporation
```
Comment (JoeOster) [inline AI-and-Analytics/End-to-end-Workloads/Census/License.txt:1]: Why the update? The (c) is not needed per legal

**Review by JoeOster — CHANGES_REQUESTED** [review]

File: DirectProgramming/C++/CompilerInfrastructure/OpenMP_Offload_Features/License.txt (hunk 1 above)
Comment (xmnboy) [reply DirectProgramming/C++/CompilerInfrastructure/OpenMP_Offload_Features/License.txt:1]: Legal has been inconsistent. Some are saying no (c) required. Some are saying (c) without a year. And some are saying (c) with first year, but no subsequent years required. So I've been going with the middle ground.

File: AI-and-Analytics/End-to-end-Workloads/Census/License.txt (hunk 2 above)
Comment (xmnboy) [reply AI-and-Analytics/End-to-end-Workloads/Census/License.txt:1]: Legal has been inconsistent. Some are saying no (c) required. Some are saying (c) without a year. And some are saying (c) with first year, but no subsequent years required. So I've been going with the middle ground.

**Comment by xmnboy** [comment]
@JoeOster -- Not clear to me how to rerun the CI test. Are you able to restart the test to see if my update to the tests for the failing sample now works?

**Review by JoeOster — APPROVED** [review]

**Pull request merged**